src/statement.cc

```diff
@@ -3,6 +3,7 @@
 #include "macros.h"
 #include "database.h"
 #include "statement.h"
+#include <napi.h>
```

Comment:
Do we need to add this.  I'm guessing it may already be in one of the include file.s